pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "mm_classification"
+version = "0.1.0"
```

Comment:
There are two sources of truth for this!  Other is `mm_classification/__init__.py`.  This is not ideal!

Reply:
changed tool.petry.name to not be slightly different

Reply:
There are still two sources of truth for `tool.poetry.version`!